Tutorials/ImaginaryTime/ising1d_imag.py

```diff
@@ -28,13 +28,13 @@
 
 # defining the hamiltonian and wrap it as matrix
 hamiltonian = nk.operator.Ising(hilbert, h=1.0)
-mat = nk.wrap_operator(hamiltonian)
+mat = nk.operator.wrap_as_matrix(hamiltonian)
 
 # create time stepper
 stepper = nk.dynamics.create_timestepper(mat.dimension, rel_tol=1e-10, abs_tol=1e-10)
 
 # prepare output
-output = nk.JsonOutputWriter('test.log', 'test.wf')
+output = nk.output.JsonOutputWriter('test.log', 'test.wf')
```

Comment:
Thanks for creating these new modules, it makes sense. I think we should also start putting together ideas for how we would like the output now, i.e. if json output is still OK or not.

Reply:
@gcarleo Agreed. Let's create an issue to discuss this there.